include/SDL3/SDL_events.h

```diff
@@ -94,6 +94,10 @@ typedef enum SDL_EventType
 
     SDL_EVENT_SYSTEM_THEME_CHANGED, /**< The system theme changed */
 
+    /* FIXME: Where do we put these? And how many should we have? */
+    SDL_EVENT_PRIVATE_RESERVED0,
+    SDL_EVENT_PRIVATE_RESERVED1,
+
```

Comment:
I think we have precedent for this in SDL_Log but I don't recall if it intentionally collides with application categories...